### ISSUE TITLE
loss

### ISSUE THREAD

**Issue by github010719** [Fediory/HVI-CIDNet#83]
代码中的loss
loss_hvi = L1_loss(output_hvi, gt_hvi) + D_loss(output_hvi, gt_hvi) + E_loss(output_hvi, gt_hvi) + opt.P_weight * P_loss(output_hvi, gt_hvi)[0]
loss_rgb = L1_loss(output_rgb, gt_rgb) + D_loss(output_rgb, gt_rgb) + E_loss(output_rgb, gt_rgb) + opt.P_weight * P_loss(output_rgb, gt_rgb)[0]
 loss = loss_rgb + opt.HVI_weight * loss_hvi

论文中的HVI的loss是l(XˆHV I , XHV I ) = λ1 · L1(XˆHV I , XHV I )  + λe · Le(XˆHV I , XHV I )  + λp · Lp(XˆHV I , XHV I )  ，我怎么觉得与代码不一致。望指正！

**Comment by Fediory** [comment]
在最新版论文中，我们已经加入了D_loss。请以最新版为准
